invenio_circulation/api.py

```diff
@@ -21,7 +21,6 @@
 class Loan(Record):
     """Loan record class."""
 
-    pid_field = "loan_pid"
     _schema = "loans/loan-v1.0.0.json"
```

Comment:
You need to refactor also all the occurrences of `Loan.pid_field` like [here](https://github.com/inveniosoftware/invenio-circulation/blob/master/invenio_circulation/pidstore/fetchers.py#L22)

Reply:
Hm, it should be included in the PR already, e.g.: https://github.com/inveniosoftware/invenio-circulation/blob/30608c0572ec708a352ae0dc566295c4a561604c/invenio_circulation/pidstore/fetchers.py